core/flow/rule.go

```diff
@@ -74,8 +74,9 @@ type FlowRule struct {
 	WarmUpPeriodSec   uint32 `json:"warmUpPeriodSec"`
 	MaxQueueingTimeMs uint32 `json:"maxQueueingTimeMs"`
 	// ClusterMode indicates whether the rule is for cluster flow control or local.
-	ClusterMode   bool              `json:"clusterMode"`
-	ClusterConfig ClusterRuleConfig `json:"clusterConfig"`
+	ClusterMode      bool              `json:"clusterMode"`
+	ClusterConfig    ClusterRuleConfig `json:"clusterConfig"`
+	WarmUpColdFactor uint32            `json:"warmUpColdFactor"`
```

Comment:
We should check the field "WarmUpColdFactor" when LoadRules.

Reply:
warmUpColdFactor  has default values. 

> func NewWarmUpTrafficShapingCalculator(warmUpPeriodInSec, warmUpColdFactor uint32, threshold float64) *WarmUpTrafficShapingCalculator {
	if warmUpColdFactor <= 1 {
		warmUpColdFactor = config.DefaultWarmUpColdFactor
	}

Reply:
Actually this should be filtered if coldFactor value is invalid.